### DISCUSSION TITLE
Cloudflare设置问题

### DISCUSSION THREAD

**Discussion by gm00039** [NewFuture/DDNS#350] [Q&A]
我在cloudflare生成了一个api key, 复制进了配置文件里。
同时我的cloudflare界面上还能看到“Zone ID”和“Account ID”，account id不是邮箱形式，两个都是一串字符串
说明里写着“也可使用Token需要列出 Zone 权限”
我需要怎么列出zone权限？

**Comment by NewFuture** [comment]
https://github.com/NewFuture/DDNS/issues/159

可以填account级的token

zone 级别的功能还没实现
